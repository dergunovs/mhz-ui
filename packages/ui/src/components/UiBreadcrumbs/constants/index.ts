export const LINKS = [
  { url: '/', title: 'Main' },
  { url: '/category', title: 'Categories' },
  { url: '/cpu', title: 'CPU' },
];

export const DEFAULT_COLOR = 'default';
export const DEFAULT_ARIA_LABEL = 'Breadcrumb';
