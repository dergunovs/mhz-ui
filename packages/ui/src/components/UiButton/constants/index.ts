export const DEFAULT_LAYOUT = 'primary';
export const DEFAULT_TYPE = 'button';
export const DEFAULT_ICON = undefined;

export const DEFAULT_SLOT = 'Button label';
export const DEFAULT_ARIA_LABEL = 'Button';
