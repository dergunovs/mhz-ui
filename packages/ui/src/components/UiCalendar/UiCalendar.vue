<template>
  <div :class="$style.container">
    <VueCal
      eventsOnMonthView
      view="month"
      :views="['month']"
      :viewsBar="false"
      :time="false"
      :events="props.events"
      :minDate="props.minDate"
      :locale="props.lang === 'en' ? 'en-us' : 'ru'"
      :todayButton="false"
      @ready="
        (dates: ICalendarReady) =>
          emit('ready', { dateFrom: dates?.view?.firstCellDate, dateTo: dates?.view?.lastCellDate })
      "
      @viewChange="
        (dates: ICalendarUpdate) => emit('update', { dateFrom: dates?.extendedStart, dateTo: dates?.extendedEnd })
      "
      @event:click="(event: ICalendarEventClick) => emit('eventClick', event?.event)"
      @cell:click="(event: ICalendarCellClick) => emit('chooseDate', event?.cell?.start)"
      data-test="ui-calendar"
    >
      <template #event="{ event }">
        <div :class="$style.title" :style="`background: ${event.color}`">
          {{ event.title }}
        </div>
      </template>
    </VueCal>
  </div>
</template>

<script setup lang="ts">
import { VueCal } from 'vue-cal';
// eslint-disable-next-line import-x/no-unassigned-import
import 'vue-cal/style';

import {
  ICalendarEvent,
  ICalendarDates,
  ICalendarReady,
  ICalendarUpdate,
  ICalendarEventClick,
  ICalendarCellClick,
} from './interface';

interface IProps {
  minDate?: Date;
  events?: ICalendarEvent<unknown>[];
  lang?: string;
}

const props = defineProps<IProps>();
const emit = defineEmits<{
  ready: [dates: ICalendarDates];
  update: [dates: ICalendarDates];
  eventClick: [event: ICalendarEvent<unknown>];
  chooseDate: [date: Date];
}>();
</script>

<style module lang="scss">
.container {
  width: 100%;

  .title {
    display: flex;
    align-items: center;
    justify-content: center;
    width: 100%;
    height: 100%;
    font-size: 1.125rem;
    font-weight: 700;
    color: var(--color-gray-light);
  }

  :global(.vuecal) {
    --vuecal-primary-color: var(--color-gray-light);
    --vuecal-secondary-color: var(--color-white);
    --vuecal-base-color: var(--color-black);
    --vuecal-contrast-color: var(--color-black);
    --vuecal-border-color: var(--color-gray);
    --vuecal-header-color: var(--color-black);
    --vuecal-event-color: var(--color-white);
    --vuecal-border-radius: 8px;
    --vuecal-height: auto;
    --vuecal-transition-duration: 0;
  }

  :global(.vuecal-event-delete-leave-active) {
    transition: 0ms;
  }

  :global(.vuecal-event-delete-leave-to) {
    transform: none;
  }

  :global(.vuecal__header) {
    border: 1px solid var(--color-gray);
  }

  :global(.vuecal__title-bar) {
    padding-top: 7px;
    padding-bottom: 6px;
    background-color: var(--color-gray-light-extra);
    border-top-left-radius: 8px;
    border-top-right-radius: 8px;
  }

  :global(.vuecal__headings) {
    height: 36px;
    padding-top: 4px;
    border-bottom: 1px solid var(--color-gray-light);
  }

  :global(.vuecal--default-theme .vuecal__weekday) {
    font-size: 1rem;
    letter-spacing: 0;
  }

  :global(.vuecal__body) {
    grid-template-rows: auto;
  }

  :global(.vuecal--default-theme .vuecal__scrollable--month-view .vuecal__cell) {
    align-items: center;
    min-height: 72px;
  }

  :global(.vuecal--default-theme:is(.vuecal--sm, .vuecal--lg) .vuecal__scrollable--month-view .vuecal__cell-date) {
    aspect-ratio: 6/5;
    padding: 2px 0 0;
    margin: 0;
    font-size: 1rem;
    font-weight: 400;
    letter-spacing: 0;
  }

  :global(
    .vuecal--default-theme.vuecal--light:is(.vuecal--sm, .vuecal--lg)
      .vuecal__scrollable--month-view
      .vuecal__cell--today
      .vuecal__cell-date
  ),
  :global(
    .vuecal--default-theme.vuecal--light:is(.vuecal--sm, .vuecal--lg)
      .vuecal__scrollable--month-view
      .vuecal__cell--selected
      .vuecal__cell-date
  ) {
    font-weight: 700;
    background-color: unset;
  }

  :global(.vuecal--default-theme .vuecal__cell) {
    box-shadow: 0 0 0 0.2px var(--color-gray) inset;
  }

  :global(.vuecal--default-theme.vuecal--timeless .vuecal__cell-events) {
    align-items: center;
    justify-content: center;
    padding-top: 0;
    padding-bottom: 7px;
  }

  :global(.vuecal__scrollable--month-view .vuecal__event) {
    width: 36px;
    height: 36px;
    padding-top: 0;
    padding-bottom: 0;
    cursor: pointer;
    border: none;
  }

  :global(.vuecal__scrollable--month-view .vuecal__event),
  :global(.vuecal--default-theme .vuecal__event-details) {
    padding: 0;
    border-radius: 12px;
  }

  :global(.vuecal--default-theme .vuecal__cell--disabled .vuecal__cell-date),
  :global(.vuecal__cell--out-of-range) {
    color: var(--color-gray-dark);
    opacity: 1;
  }

  :global(.vuecal--default-theme .vuecal__weekday-day) {
    opacity: 1;
  }
}
</style>
