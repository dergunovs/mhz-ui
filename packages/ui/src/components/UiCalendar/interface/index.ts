export interface ICalendarEvent<T> {
  id?: string;
  start: Date | null;
  end: Date | null;
  title: string;
  content: T[];
  color?: string;
}

export interface ICalendarUpdate {
  firstCellDate: string;
  lastCellDate: string;
}
