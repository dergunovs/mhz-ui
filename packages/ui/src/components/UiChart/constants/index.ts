export const LABELS = [
  '07.07 - 13.07',
  '14.07 - 20.07',
  '21.07 - 27.07',
  '28.07 - 03.08',
  '04.08 - 10.08',
  '11.08 - 17.08',
  '18.08 - 24.08',
];

export const DATASETS = [
  {
    data: [28, 16, 12, 4, 22, 16, 14],
    label: 'Грудь',
    borderColor: '#00CED1',
    backgroundColor: '#00CED1',
  },
  {
    data: [4, 6, 0, 2, 2, 0, 6],
    label: 'Ноги',
    borderColor: '#000080',
    backgroundColor: '#000080',
  },
  {
    data: [22, 18, 8, 10, 25, 18, 16],
    label: 'Плечи',
    borderColor: '#C41E3A',
    backgroundColor: '#C41E3A',
  },
  {
    data: [0, 6, 0, 6, 5, 4, 2],
    label: 'Пресс',
    borderColor: '#32CD32',
    backgroundColor: '#32CD32',
  },
  {
    data: [32, 24, 12, 10, 33, 24, 20],
    label: 'Руки',
    borderColor: '#DA70D6',
    backgroundColor: '#DA70D6',
  },
  {
    data: [16, 18, 8, 12, 26, 22, 20],
    label: 'Спина',
    borderColor: '#FF8C00',
    backgroundColor: '#FF8C00',
  },
];

export const TYPE = 'Line';
export const TITLE = 'Chart title';
