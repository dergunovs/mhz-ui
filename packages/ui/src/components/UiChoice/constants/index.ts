export const options = [1, 2, 3, 4, 5];
export const modelValue = options[2];
export const title = 'Заголовок';
export const isTall = true;
export const isInput = true;
