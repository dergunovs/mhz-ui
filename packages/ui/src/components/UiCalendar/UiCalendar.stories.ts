import { ref } from 'vue';
import { Meta, StoryObj } from '@storybook/vue3';
import { html } from 'mhz-helpers';

import { EVENTS, LANG } from './constants';
import { ICalendarEvent } from './interface';

import { UiCalendar, UiModal } from '@/components';

const meta: Meta<typeof UiCalendar> = {
  component: UiCalendar,
  args: {
    events: EVENTS,
    lang: LANG,
  },
  argTypes: {
    lang: {
      options: ['en', 'ru'],
    },
  },
};

export default meta;

export const Primary: StoryObj<typeof UiCalendar> = {
  render: (args) => ({
    components: { UiCalendar, UiModal },
    setup: () => {
      const isShowModal = ref(false);
      const eventContent = ref();

      function handleEventClick(event: ICalendarEvent<object>) {
        eventContent.value = event.content;
        isShowModal.value = !isShowModal.value;
      }

      return { args, isShowModal, handleEventClick, eventContent };
    },

    template: html` <UiCalendar v-bind="args" @eventClick="handleEventClick" />
      <UiModal v-model="isShowModal">{{eventContent}}</UiModal>`,
  }),
};
