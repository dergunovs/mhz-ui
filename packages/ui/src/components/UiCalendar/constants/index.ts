import { ICalendarEvent } from '../interface';

export const EVENTS: ICalendarEvent<object>[] = [
  {
    id: 'id of event',
    start: new Date(),
    end: new Date(),
    title: '+',
    content: [{ id: 1, text: 'Text' }],
    color: 'linear-gradient(135deg, rgb(168, 60, 255) 36%, rgb(235, 0, 123) 69%, rgb(251, 185, 0) 100%)',
  },
  {
    id: 'id of event',
    start: new Date(),
    end: new Date(),
    title: '+',
    content: [{ id: 2, text: 'Text 2' }],
    color: 'linear-gradient(135deg, rgb(168, 60, 255) 41%, rgb(235, 0, 123) 71%, rgb(0, 134, 255) 100%)',
  },
];
