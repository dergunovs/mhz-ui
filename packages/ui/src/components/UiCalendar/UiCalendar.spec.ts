import { DefineComponent, nextTick } from 'vue';
import { describe, it, expect, beforeEach, afterEach } from 'vitest';
import { VueWrapper, enableAutoUnmount } from '@vue/test-utils';
import { dataTest } from 'mhz-helpers';

import UiCalendar from './UiCalendar.vue';

import { EVENTS, LANG } from './constants';
import { ICalendarEvent } from './interface';
import { wrapperFactory } from '@/test';

const calendar = dataTest('ui-calendar');

let wrapper: VueWrapper<InstanceType<typeof UiCalendar>>;

beforeEach(() => {
  wrapper = wrapperFactory(UiCalendar, {
    events: EVENTS,
    lang: LANG,
  });
});

enableAutoUnmount(afterEach);

describe('UiCalendar', async () => {
  it('exists', async () => {
    expect(wrapper.findComponent(UiCalendar)).toBeTruthy();
  });

  it('matches snapshot', async () => {
    expect(wrapper.html()).toMatchSnapshot();
  });

  it('sets events to calendar', async () => {
    expect(
      wrapper.findComponent<DefineComponent<{ events: ICalendarEvent<unknown>[] }>>(calendar).props('events')
    ).toStrictEqual(EVENTS);
  });

  it('emits calendar events', async () => {
    const date = new Date();
    const date2 = new Date();

    const formattedDates = { dateFrom: date, dateTo: date2 };

    const readyEvent = { view: { firstCellDate: date, lastCellDate: date2 } };
    const updateEvent = { extendedStart: date, extendedEnd: date2 };
    const clickEvent = { event: EVENTS[0] };
    const cellEvent = { cell: { start: date } };

    wrapper.findComponent<DefineComponent>(calendar).vm.$emit('ready', readyEvent);
    wrapper.findComponent<DefineComponent>(calendar).vm.$emit('viewChange', updateEvent);
    wrapper.findComponent<DefineComponent>(calendar).vm.$emit('event:click', clickEvent);
    wrapper.findComponent<DefineComponent>(calendar).vm.$emit('cell:click', cellEvent);

    await nextTick();

    expect(wrapper.emitted('ready')).toHaveLength(1);
    expect(wrapper.emitted()['ready'][0]).toEqual([formattedDates]);

    expect(wrapper.emitted('update')).toHaveLength(1);
    expect(wrapper.emitted()['update'][0]).toEqual([formattedDates]);

    expect(wrapper.emitted('eventClick')).toHaveLength(1);
    expect(wrapper.emitted()['eventClick'][0]).toEqual([EVENTS[0]]);

    expect(wrapper.emitted('chooseDate')).toHaveLength(1);
    expect(wrapper.emitted()['chooseDate'][0]).toEqual([date]);
  });

  it('emits ready event with correct date format', async () => {
    const date = new Date();
    const date2 = new Date();

    const readyEvent = { view: { firstCellDate: date, lastCellDate: date2 } };

    wrapper.findComponent<DefineComponent>(calendar).vm.$emit('ready', readyEvent);

    await nextTick();

    expect(wrapper.emitted('ready')).toHaveLength(1);
    expect(wrapper.emitted()['ready'][0]).toEqual([{ dateFrom: date, dateTo: date2 }]);
  });

  it('emits update event with correct date format', async () => {
    const date = new Date();
    const date2 = new Date();

    const updateEvent = { extendedStart: date, extendedEnd: date2 };

    wrapper.findComponent<DefineComponent>(calendar).vm.$emit('viewChange', updateEvent);

    await nextTick();

    expect(wrapper.emitted('update')).toHaveLength(1);
    expect(wrapper.emitted()['update'][0]).toEqual([{ dateFrom: date, dateTo: date2 }]);
  });

  it('emits eventClick with correct event data', async () => {
    const clickEvent = { event: EVENTS[0] };

    wrapper.findComponent<DefineComponent>(calendar).vm.$emit('event:click', clickEvent);

    await nextTick();

    expect(wrapper.emitted('eventClick')).toHaveLength(1);
    expect(wrapper.emitted()['eventClick'][0]).toEqual([EVENTS[0]]);
  });

  it('emits chooseDate with correct date', async () => {
    const date = new Date();
    const cellEvent = { cell: { start: date } };

    wrapper.findComponent<DefineComponent>(calendar).vm.$emit('cell:click', cellEvent);

    await nextTick();

    expect(wrapper.emitted('chooseDate')).toHaveLength(1);
    expect(wrapper.emitted()['chooseDate'][0]).toEqual([date]);
  });
});
