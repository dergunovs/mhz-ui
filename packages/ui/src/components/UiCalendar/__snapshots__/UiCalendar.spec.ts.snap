// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`UiCalendar > matches snapshot 1`] = `
"<div class="_container_4b457f">
  <div class="_header_4b457f">
    <div class="_titleBar_4b457f"><button class="_navButton_4b457f" type="button" data-test="ui-calendar-prev-month">&lt;</button><span data-test="ui-calendar-current-month">Октябрь 2025</span><button class="_navButton_4b457f" type="button" data-test="ui-calendar-next-month">&gt;</button></div>
    <div class="_headings_4b457f">
      <div class="_weekday_4b457f" data-test="ui-calendar-week-day">пн</div>
      <div class="_weekday_4b457f" data-test="ui-calendar-week-day">вт</div>
      <div class="_weekday_4b457f" data-test="ui-calendar-week-day">ср</div>
      <div class="_weekday_4b457f" data-test="ui-calendar-week-day">чт</div>
      <div class="_weekday_4b457f" data-test="ui-calendar-week-day">пт</div>
      <div class="_weekday_4b457f" data-test="ui-calendar-week-day">сб</div>
      <div class="_weekday_4b457f" data-test="ui-calendar-week-day">вс</div>
    </div>
  </div>
  <div class="_body_4b457f">
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">29</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">30</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">1</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">2</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">3</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">4</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">5</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">6</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">7</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">8</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">9</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">10</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">11</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">12</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">13</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">14</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">15</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">16</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">17</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">18</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">19</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">20</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">21</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f _today_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">22</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">23</div>
      <div class="_cellEvents_4b457f">
        <div class="_event_4b457f" data-test="ui-calendar-event">
          <div class="_title_4b457f" data-test="ui-calendar-event-title">24</div>
        </div>
        <div class="_event_4b457f" data-test="ui-calendar-event">
          <div class="_title_4b457f" data-test="ui-calendar-event-title">32</div>
        </div>
      </div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">24</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">25</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">26</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">27</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">28</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">29</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">30</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">31</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">1</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
    <div class="_cell_4b457f" data-test="ui-calendar-calendar-day">
      <div class="_cellDate_4b457f" data-test="ui-calendar-cell-date">2</div>
      <div class="_cellEvents_4b457f"></div>
    </div>
  </div>
</div>"
`;
